src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -72,10 +72,10 @@ class ActiveTabGlobalTrackComponent extends PureComponent<Props> {
     const { globalTrack } = this.props;
     switch (globalTrack.type) {
       case 'tab': {
-        const { mainThreadIndex } = globalTrack;
+        const { threadsKey } = globalTrack;
         return (
           <TimelineTrackThread
-            threadsKey={mainThreadIndex}
+            threadsKey={threadsKey}
```

Comment:
We should probably capture the knowledge about overlapping threads in an issue, since this is now introducing that.